### DISCUSSION TITLE
Frontend deployment region

### DISCUSSION THREAD

**Discussion by markkkkas** [vercel/community#702] [Help] [answered]
Hey,

recently i started to use Vercel and asked myself in which region my frontend is deployed? I know that i can select region for my serverless functions (e.g fra1, arn1, iad1, ...), but can I select region for my frontend or it will be adjusted to closest location which user is?

Thanks in advance!

**Comment by leerob** [comment, marked as answer]
Your frontend (e.g. HTML, JS, CSS) will be automatically deployed to every region in Vercel's Edge Network. Then, your functions will run in the region defined in your project settings (or in your `vercel.json` file, if it exists).

**Reply by markkkkas** [reply]
Thank you!
